lib/latinize.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for latinize 0.2.0
+// Project: https://github.com/dundalek/latinize
+// Definitions by: Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace LatinizeModule {
+  export interface Latinize {
+    (str: string): string;
+    characters: { [char: string]: string };
+  }
+}
+
+declare var latinize: LatinizeModule.Latinize;
```

Comment:
From what I can see you're using imports to use latinize, so the definition does not have to be global here.